action.yml

```diff
@@ -1,12 +1,22 @@
-name: 'Project recognition'
-description: 'Allocate labels depends on changed files'
+name: 'Recognition'
+description: 'Recognizes affected area based on changed files'
 inputs:
-  label_filename:
-    description: 'Filename which contain label metadata to look for'
-    required: true
+  matrix:
+    description: 'Entities to be recognized under label source, used as output field in matrix. JSON string.'
+    required: false
+  labels:
+    description: 'Defines whether output labels'
+    required: false
+    default: 'false'
   token:
     description: 'GitHub token'
     required: true
+  source:
+    description: 'Label source filename'
+    required: true
```

Comment:
mmmm I think you completely forgot the reviewers branch when you renamed this, we're gonna have also the all reviewers saga coming up (was supposed to come today but now will have to push it to v3 since merging these two together is gonna be a hassle.
There is a source for labels and a source for owners you can't call one of them `source`